WalletWasabi/Helpers/Constants.cs

```diff
@@ -21,6 +21,7 @@ public static class Constants
 		public const uint ProtocolVersionWitnessVersion = 70012;
 
 		public const int P2wpkhInputSizeInBytes = 41;
+		public const int P2wpkhInputVirtualSizeAccordingBtcPayServer = 68;
 		public const int P2pkhInputSizeInBytes = 145;
 		public const int OutputSizeInBytes = 33;
```

Comment:
Shouldn't these then be 148, 31 respectively?

Reply:
out of scope.